src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -882,20 +802,15 @@ The dry potential temperature, given a thermodynamic state `ts`.
 """
 dry_pottemp(ts::ThermodynamicState) =
   dry_pottemp(air_temperature(ts), air_density(ts), PhasePartition(ts))
-dry_pottemp(ts::PhaseDry) = air_temperature(ts) / exner(ts)
 
 """
     air_temperature_from_liquid_ice_pottemp(θ_liq_ice, ρ, q::PhasePartition)
 
-Air temperature
-``
-  T = θ_{liq-ice}*(ρ*R_m*θ_{liq-ice}/MSLP)^{R_m/c_{vm}} + latent_heat_liq_ice(q)/c_{vm}
-``
-given
+The temperature given
  - `θ_liq_ice` liquid-ice potential temperature
  - `ρ` (moist-)air density
 and, optionally,
- - `q` [`PhasePartition`](@ref). Without this argument the results are that of dry air.
+ - `q` [`PhasePartition`](@ref). Without this argument, the results are for dry air.
```

Comment:
Please add the quadratic term from the design docs in l. 820.

Reply:
Raised issue #567